### PR TITLE
[newa] lexical model with ~21k words

### PR DESCRIPTION
**Pull request by sapradhan**
this is a wordlist for newa (nepal bhasa). it has about 21k words without word frequency

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+(cc) 2020 Ananda K Maharjan, Santosh Pradhan
```
Comment (darcywong00) [inline release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md:3]:
```suggestion
(c) 2020 Ananda K Maharjan, Santosh Pradhan
```

File: release/sapradhan/sapradhan.new-newa.nepal_bhasa/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+nepal-bhasa lexical model
+===================
+
+(cc) 2020 Ananda K Maharjan, Santosh Pradhan
```
Comment (darcywong00) [inline release/sapradhan/sapradhan.new-newa.nepal_bhasa/README.md:4]:
```suggestion
(c) 2020 Ananda K Maharjan, Santosh Pradhan
```

File: release/sapradhan/sapradhan.new-newa.nepal_bhasa/source/readme.htm [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+</head>
+<body>
+
+<h1>nepal-bhasa</h1>
+
+<p>
+    Contains Nepal Bhasa words in Newa script.
+</p>
+
+<p>(cc) 2020 Ananda K Maharjan, Santosh Pradhan</p>
```
Comment (darcywong00) [inline release/sapradhan/sapradhan.new-newa.nepal_bhasa/source/readme.htm:20]:
```suggestion
<p>(c) 2020 Ananda K Maharjan, Santosh Pradhan</p>
```

File: release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md (hunk 1 above)
Comment (sapradhan) [reply release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md:3]: can it be left to creative commons license ?

Comment (darcywong00) [reply release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md:3]: We chose the MIT license in the lexical-models repo to follow the Keyman keyboards repo. 

See the bottom of this page for more info (Assigning an open source license):
https://help.keyman.com/developer/keyboards/advanced/step-0

Comment (sapradhan) [reply release/sapradhan/sapradhan.new-newa.nepal_bhasa/LICENSE.md]: changed as requested

**Pull request merged**